appsec/config.go

```diff
@@ -18,22 +18,34 @@ import (
 
 // Configuration environment variables
 const (
-	envAPISecEnabled    = "DD_EXPERIMENTAL_API_SECURITY_ENABLED"
-	envAPISecSampleRate = "DD_API_SECURITY_REQUEST_SAMPLE_RATE"
-	envObfuscatorKey    = "DD_APPSEC_OBFUSCATION_PARAMETER_KEY_REGEXP"
-	envObfuscatorValue  = "DD_APPSEC_OBFUSCATION_PARAMETER_VALUE_REGEXP"
-	envWafTimeout       = "DD_APPSEC_WAF_TIMEOUT"
-	envTraceRateLimit   = "DD_APPSEC_TRACE_RATE_LIMIT"
-	envRules            = "DD_APPSEC_RULES"
+	// EnvAPISecEnabled is the env var used to enable API Security
+	EnvAPISecEnabled = "DD_EXPERIMENTAL_API_SECURITY_ENABLED"
```

Comment:
I guess the whole `_EXPERIMENTAL_` will need dropping at some point... The question now is - should the constant name include `Experimental` or not? Supposedly, when API Sec goes out of beta, there'll at least be some window during which you'll want the `_EXPERIMENTAL_` and "definitive" variables to both be honored?

Reply:
I think it's fine not to translate the `_EXPERIMENTAL_` bit in the const name since it's purpose is to make sure users know what they sign-up for when using the env var, without changing anything to the feature.
As for honouring both, I don't think that's an issue either since the swap between the two will happen over a tracer release. We'll switch from one two the other, I don't think we'd need to handle both in a same version.